### PR TITLE
Ne pas afficher presta injectees

### PR DESCRIPTION
**Pull request by fpagnoux**
Connected to sgmap/mes-aides-ui#118

Similar to reverted #35  with a small bug fix. 

Relies on sgmap/mes-aides-ui#136

### CODE REVIEW

**Comment by MattiSG** [comment]
GTM & good catch!

File: lib/simulation/openfisca/mapping/reverse.js [hunk 1]
```diff
@@ -4,10 +4,17 @@ var periods = require('./periods');
 var PRESTATIONS = require('../prestations');
 
 
-module.exports = function reverseMap(openFiscaFamille, date) {
+module.exports = function reverseMap(openFiscaFamille, date, injectedRessources) {
```
Comment (MattiSG) [inline lib/simulation/openfisca/mapping/reverse.js]: This is getting really hard to grasp. Add a [JSdoc comment](http://usejsdoc.org/tags-param.html) to describe the parameters of this function.

File: lib/simulation/openfisca/mapping/reverse.js [hunk 2]
```diff
@@ -4,10 +4,17 @@ var periods = require('./periods');
...
     var period = periods.map(date);
+    var injectedPrestations = _.intersection(_.keys(PRESTATIONS), injectedRessources)
 
     return _.mapValues(PRESTATIONS, function(format, prestationName) {
+
+    // For prestations that have been injected by the user and not calculated by the simulator, put null to allow specific treatment in the ui
```
Comment (MattiSG) [inline lib/simulation/openfisca/mapping/reverse.js]: Missing identation.

File: lib/simulation/openfisca/mapping/reverse.js (hunk 1 above)
Comment (fpagnoux) [inline lib/simulation/openfisca/mapping/reverse.js]: I reorganized the two functions, giving to reverseMap the responsibility to compute the injected resources. Is it more clear ?

Comment (MattiSG) [inline lib/simulation/openfisca/mapping/reverse.js]: Perfect, much better response than adding doc!

**Pull request merged**